backend/Squirrel.ConsoleApp.BL/Services/FilePathHelperService.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Squirrel.ConsoleApp.BL.Services;
+
+public static class FilePathHelperService
+{
+    public static string GetDbSettingsFilePath()
+    {
+        return Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "squirrel-db-settings.json");
```

Comment:
Environment... - to private function

Reply:
What could I can use instead? I thought that save user settings in UserProfile directory - it's ok...

Reply:
Done